miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileDownloader.kt

```diff
@@ -165,16 +166,21 @@ class MiniAppFileDownloaderDefault(var activity: Activity, var requestCode: Int)
         else
             ""
         val mimetype = MimeTypeMap.getSingleton().getMimeTypeFromExtension(extension)
-        return if (!mimetype.isNullOrBlank())
-            mimetype
-        else
-            "text/plain"
+
+        return if (Build.VERSION.SDK_INT == Build.VERSION_CODES.Q) {
```

Comment:
(optional) add some remarks as to why to avoid confusion in the future

Reply:
Yeah, I've added a note for this use-case 🙇‍♂️